### PR TITLE
Removes HRData temp rule

### PR DESCRIPTION
**Pull request by dividehex**
Removes the HRData rule

### CODE REVIEW

**Comment by gcoxmoz** [comment]
https://mozilla-hub.atlassian.net/browse/IAM-1068

File: rules/temporary-update-at-conformance.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ function temporaryUpdateAtConformance(user, context, callback) {
     'aDL5o9SZRaYTH5zzkGntT4l76qydMbZe', // sso dashboard allizom
     'UCOY390lYDxgj5rU8EeXRtN6EP005k7V', // sso dashboard prod
     'mc1l0G4sJI2eQfdWxqgVNcRAD9EAgHib', // sso dashboard allizom
+    '2KNOUCxN8AFnGGjDCGtqiDIzq8MKXi2h', // sso dashboard allizom
```
Comment (floatingatoll) [inline rules/temporary-update-at-conformance.js:7]: This change snuck in, in case it's meant to be in a separate PR.

**Review by floatingatoll — APPROVED** [review]
Looks fine, one note.

File: rules/temporary-update-at-conformance.js (hunk 1 above)
Comment (dividehex) [reply rules/temporary-update-at-conformance.js:7]: Thanks for noticing.  This was intentional since it was a change I had queued up but hadn't made a PR for it.  So it got swept into this one.
